nylas/client/client.py

```diff
@@ -466,11 +474,13 @@ def _get_resources(self, cls, extra=None, **filters):
         return [cls.create(self, **x) for x in results if x is not None]
 
     def _get_resource_raw(
-        self, cls, id, extra=None, headers=None, stream=False, **filters
+        self, cls, id, extra=None, headers=None, stream=False, path=None, **filters
     ):
         """Get an individual REST resource"""
+        if path is None:
+            path = cls.collection_name
         postfix = "/{}".format(extra) if extra else ""
-        path = "/{}".format(cls.collection_name) if cls.collection_name else ""
+        path = "/{}".format(path) if path else ""
```

Comment:
what would a false-y `path` look like here if you're setting it at 481?

Reply:
If `path` isn't passed in, path is set to `cls.collection_name`. This is just on the off chance that the `collection_name` isn't set for the collection passed in.